packages/cacti-copm-core/src/main/json/openapi.json

```diff
@@ -160,28 +165,16 @@
       },
       "AssetLockClaimV1": {
         "description": "Claim a previously locked asset on the same network.\n\"lock_id\" is required for fungible assets, \"asset\" is required for non-fungible assets.\nhttps://github.com/hyperledger-cacti/cacti/blob/main/weaver/rfcs/protocols/asset-exchange/generic-htlc.md",
-        "required": [
-          "lock_id",
-          "source",
-          "destination",
-          "asset",
-          "source_certificate",
-          "hash_info"
-        ],
+        "required": ["lock_id", "destination", "asset", "hash_info"],
```

Comment:
Why remove `source` (or `source_certificate`)?

Reply:
I changed the lock claim for the fabric NFT case to use the lock id instead of the asset agreement to claim the lock, to be consistent with the Corda implementation.  After making this change you don't need information about source any more.